### PR TITLE
fix: mailer template issue

### PR DESCRIPTION
**Pull request by balub**
Closes HBE-304

### Description

In this PR, we fix a known issue with the mailer module where the `templates` folder is not present in the `dist` folder after the image has been built.

### Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### Testing info
Make sure to build the image using the following command, with the necessary changes.
```
docker build -t hoppscotch-local-aio -f prod.Dockerfile --target aio .
```
 Check if the following type of emails are being sent correctly using `mailcatcher`:
- User-Invitations from the admin app.
- Team-Invitation for the app.

### CODE REVIEW

**Review by mirarifhasan — APPROVED** [review]
Works for me

File: packages/hoppscotch-backend/src/templates/code-your-own.hbs
Comment (AndrewBastin) [inline packages/hoppscotch-backend/src/templates/code-your-own.hbs:1]: can this file be named to something else, as the name doesn't reflect what the file template contains ?

**Review by AndrewBastin — CHANGES_REQUESTED** [review]

File: packages/hoppscotch-backend/src/templates/code-your-own.hbs
Comment (balub) [reply packages/hoppscotch-backend/src/templates/code-your-own.hbs:1]: Sure

**Review by AndrewBastin — APPROVED** [review]
lgtm :100:

**Pull request merged**